packages/aws-cdk-lib/core/lib/fs/index.ts

```diff
@@ -26,10 +26,11 @@ export class FileSystem {
   /**
    * Produces fingerprint based on the contents of a single file or an entire directory tree.
    *
+   * Line endings are converted from CRLF to LF.
+   *
    * The fingerprint will also include:
    * 1. An extra string if defined in `options.extra`.
-   * 2. The set of exclude patterns, if defined in `options.exclude`
-   * 3. The symlink follow mode value.
+   * 2. The symlink follow mode value.
    *
    * @param fileOrDirectory The directory or file to fingerprint
    * @param options Fingerprinting options
```

Comment:
The original doc is [here](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/core/lib/fs/fingerprint.ts#L28-L39).

```ts
/**
 * Produces fingerprint based on the contents of a single file or an entire directory tree.
 *
 * Line endings are converted from CRLF to LF.
 *
 * The fingerprint will also include:
 * 1. An extra string if defined in `options.extra`.
 * 2. The symlink follow mode value.
 *
 * @param fileOrDirectory The directory or file to fingerprint
 * @param options Fingerprinting options
 */
export function fingerprint(fileOrDirectory: string, options: FingerprintOptions = { }) {
```